crates/allocator/src/bump.rs

```diff
@@ -321,28 +314,6 @@ mod fuzz_tests {
         TestResult::passed()
     }
 
-    #[quickcheck]
-    fn should_not_allocate_arbitrary_bytes_if_they_overflow(n: usize) -> TestResult {
```

Comment:
Could you shortly explain why you think the test is no longer necessary?

Reply:
I explain in the commit which removed it (https://github.com/paritytech/ink/pull/1336/commits/fe85b7e93afc55c91d1488e58ec61ca52b7e4e1c) 😉